src/cli-options.ts

```diff
@@ -243,6 +243,12 @@ function getArgvOptions(
       //     though it is slightly different to the npm 6 version.
       return parseRemainingArgs(findRemainingArgsFromNpmConfigArgv(script));
     }
+    case 'yarnBerry': {
+      // yarn 3.2.1
+      //   - argv contains the path the to node, the path to wireit, and each
+      //     argument
+      return parseRemainingArgs(process.argv.slice(2));
+    }
     case 'pnpm': {
       // pnpm 7.1.7
       //   - Arguments before the script name are pnpm arguments and error if unknown.
```

Comment:
```suggestion
    }
    case 'yarnBerry':    
    case 'pnpm': {
      // yarn 3.2.1
      //   - Arguments before the script name are yarn arguments and error if unknown. 
      //   - Arguments after the script name go to argv.
      // pnpm 7.1.7
      //   - Arguments before the script name are pnpm arguments and error if unknown.
```

Since it's identical behavior to pnpm (I just verified the first thing, about flags before the script name).